src/fonduer/learning/disc_models/layers/rnn.py

```diff
@@ -7,6 +7,28 @@
 
 
 class RNN(nn.Module):
+    """A recurrent neural network layer.
+
+    :param n_classes: Number of classes.
```

Comment:
Consider renaming to `num_classes` for consistency with `num_tokens`, `num_layers`, etc.